docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -386,9 +274,9 @@ The following table shows a comparison of capabilities supported by {beats} and
 
 |Multiple outputs
 |{y}
-|{n}
 |{y}
-|{fleet}-managed {agent}s only provide a <<elastic-agent-output-configuration,single global output>> to the same {es} cluster where {fleet} is running. We are considering support for https://github.com/elastic/kibana/issues/104980[more outputs].
+|{y}
+|A single {fleet}-managed {agent}'s policy can be configured to have multiple outputs if desired.
```

Comment:
Flipped the sentence around a little because I think the adjective string makes the sentence a little harder to scan.

Also...I think "if desired" is not necessary.

Also...about the "multiple outputs" statement. By multiple outputs, do you mean that the policy can send monitoring data and collected data to different outputs? I think saying multiple outputs might raise expectations.